README.md

```diff
@@ -108,13 +108,7 @@ After generating your password, you can find the following attributes in the con
 - Role
 - Plain Text (password)
 
-You can derive your connection string with the following format.
-
-```text
-mysql://<USERNAME>:<PLAIN_TEXT_PASSWORD>@<ACCESS_HOST_URL>/<DATABASE_NAME>?ssl=true
-```
-
-Make a copy of the `.env.example` file as `.env` and update the `DATABASE_URL` property with your connection string.
+Make a copy of the `.env.example` file as `.env` and update the `DATABASE_HOST`, `DATABASE_NAME`, `DATABASE_USERNAME`, and `DATABASE_PASSWORD` properties using the properties of the password you just generated.
```

Comment:
> properties using the properties

can we swap one of the "properties" out? maybe say values instead for the first?